app/routes/lending/components/LendableObjectEdit.tsx

```diff
@@ -0,0 +1,176 @@
+import { Content } from 'app/components/Content';
+import { FlexRow } from 'app/components/FlexBox';
+import {
+  Button,
+  EditorField,
+  Form,
+  SelectInput,
+  SubmitButton,
+  TextInput,
+} from 'app/components/Form';
+import LegoFinalForm from 'app/components/Form/LegoFinalForm';
+import SubmissionError from 'app/components/Form/SubmissionError';
+import { selectAllGroups } from 'app/reducers/groups';
+import { selectLendableObjectById } from 'app/reducers/lendableObjects';
+import { useAppDispatch, useAppSelector } from 'app/store/hooks';
+import { roleOptions } from 'app/utils/constants';
+import type { EditingLendableObject } from 'app/store/models/LendableObject';
+
+type Params = {
+  lendableObjectId: string | undefined;
+};
+
+const TypedLegoForm = LegoFinalForm<EditingLendableObject>;
+
+const LendableObjectEdit = () => {
+  const { lendableObjectId } = useParams<Params>();
+  const isNew = lendableObjectId === undefined;
+
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  usePreparedEffect(
+    'fetchLendableObject',
+    () => dispatch(fetchLendableObject(Number(lendableObjectId))),
+    [isNew],
+  );
+
+  const lendableObject = useAppSelector((state) =>
+    selectLendableObjectById(state, lendableObjectId),
+  );
+
+  const groups = useAppSelector(selectAllGroups);
+
+  const onSubmit = (values: EditingLendableObject) => {
+    if (isNew) {
+      dispatch(
+        createLendableObject({
+          ...values,
+          responsibleGroups: values.responsibleGroups?.map((group) => group.id),
+          responsibleRoles: values.responsibleRoles?.map((role) => role.value),
+        }),
+      ).then(() => navigate('/lending'));
+    } else {
+      dispatch(
+        editLendableObject({
+          ...values,
+          id: lendableObjectId,
+          responsibleGroups: values.responsibleGroups.map(
+            (group) => group.id || group.value,
+          ),
+          responsibleRoles: values.responsibleRoles.map((role) => role.value),
+        }),
+      ).then(() => navigate(`/lending/${lendableObjectId}`));
+    }
+  };
+
+  const onDelete = async () => {
+    if (!lendableObjectId) {
+      return;
+    }
+    await dispatch(deleteLendableObject(lendableObjectId));
+    navigate('/lending');
+  };
+
+  const initialValues = !isNew
+    ? {
+        ...lendableObject,
+        responsibleRoles: lendableObject?.responsibleRoles.map((role) => ({
+          label: roleOptions.find((r) => r.value === role)?.label || role,
+          value: role,
+        })),
+        responsibleGroups: (lendableObject?.responsibleGroups || [])
+          .filter(Boolean)
+          .map((group) => ({
+            label: groups.find((g) => g.id === group)?.name || group,
+            value: groups.find((g) => g.id === group),
+          })),
+      }
+    : {};
+
+  const showSkeleton = !(isNew || (lendableObject && groups));
+  return (
+    <Content skeleton={showSkeleton}>
```

Comment:
I believe the skeleton for `Content` is only used for the banner or something. I don't think it applies here